examples/gfx/opengl/4_opengl_bexel/bebone.h

```diff
@@ -1,8 +1,8 @@
 #ifndef BEBONE_H
 #define BEBONE_H
 
-#define OMNI_TYPES_MATRIX_COLLUM_MAJOR_ORDER
-// #define OMNI_TYPES_MATRIX4X4_PROJECTION_MATRIX_INVERSE_Y_AXIS
+#define BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER
```

Comment:
Grammar error
Change `BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER` to `BEBONE_TYPES_MATRIX_COLUMN_MAJOR_ORDER`

Reply:
yes sir !